src/scss/constants.scss

```diff
@@ -90,6 +90,7 @@ $z-layers: (
   // Map
   map-room-hovered: 3,
   map-room: 2,
+  map-seat: 1,
```

Comment:
The way I envisioned this was that we would have the `map-background` at the lowest level (which may not need an explicit `z-index`, not sure), then the `map-room-unclickable`, then `map-grid`, then `map-room` and finally `map-hovered`. Or something similar to that.

As currently defined here, it seems like we're explicitly moving the grid below the rooms. But unsure how that will impact the rest of the 'layering' as set out in my 'bug report' in https://github.com/sparkletown/internal-sparkle-issues/issues/387#issuecomment-792289425

Reply:
Further to my questions in https://github.com/sparkletown/sparkle/pull/1225#issuecomment-796404316 I have just looked at the prod environment/venue for this event, and it seems that there is a need for the unclickable room gifs to be able to be 'layered' on top of existing rooms as well.

So the changes here probably aren't going to be desirable.

Reply:
See screenshot/link in https://github.com/sparkletown/internal-sparkle-issues/issues/387#issuecomment-796474043

Reply:
So, it should be over the grid? I don't see other way to have it over the other rooms. So basically, we should revert the latest changes. Is that what you mean?

Reply:
I replied on slack before seeing this, copying here for reference:

> @0xdevalias:
>
> The request is the ability to arbitrarily assign z-indexes to each unclickable room individually, so that their layering can be precisely controlled individually; not as an entire ‘block of things’ on a single layer.
> 
> Some unclickable rooms want to be at the ‘map background image’ layer (which I feel should probably be our default), some want to be sitting ‘above’ every other layer (eg. the birds flying around the top of one of the island room images)
> 
> Eventually we would want to expose this ability to ‘reorder room layers’ in the admin UI tooling directly, but until then, it’s probably sufficient to follow a similar pattern as we did for the ‘iframe venue’, and allow the z-index to be set within the database on the room objects, then used as inline styles.
> <sub>[View in Slack](https://doesitsparkle.slack.com/archives/C01JHLMSFND/p1615499612333000)</sub>

Reply:
Can you rename this `map-grid` or something similar please.

Reply:
Can you rename this `map-grid` or something similar please.

Reply:
did in https://github.com/sparkletown/sparkle/pull/1225/commits/c780464ddb4431dccdb1bef05448301e56ee1c9a